ballroom-beats/src/components/Navbar/Navbar.js

```diff
@@ -1,32 +1,53 @@
 import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { createAppContainer, withNavigation  } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
-import { withNavigation } from 'react-navigation';
+import { StyleSheet, View, Text, Dimensions } from 'react-native';
+import Icon from 'react-native-vector-icons/AntDesign';
 
-class Navbar extends Component {
 
+class Navbar extends Component {
+  
   render() {
-      return (
-          <View style={styles.container}>
-            <Text>Navbar</Text>
-          </View>
-      )      
-  }
-}; 
+    return (
+      <View style={styles.bottomContainer}>
+          <View style={styles.btnContainer}>
+            <Icon.Button 
+                style={styles.button} 
+                name="home" 
+                onPress={() => this.props.navigation.navigate('Home')} 
+                title="BACK">
+                <Text style={styles.home}>Home</Text>
+            </Icon.Button>
+      </View>
+    </View>
+  )
```

Comment:
@spaceplesiosaur Hi Allie, there should be blue navbar at the bottom of DanceType, SongPick and Loader that will return the user back to the Home component